dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -1698,9 +1698,10 @@ def test_diff(self):
             },
             "t-test": {
                 "t-statistic": 0.5393164101529813,
-                "conservative": {"df": 2, "p-value": 0.643676756587475},
+                "conservative": {"df": 2.0, "p-value": 0.643676756587475},
```

Comment:
`int` I believe

Reply:
I just checked L561 in `numerical_column_stats.py` and it shows the below... So, I think we are good with `1.0`?

```
results["conservative"]["df"] = float(conservative_df)
```